### PR TITLE
Add multifile support to quicinterop

### PR DESCRIPTION
**Pull request by anrossi**

### CODE REVIEW

File: src/tools/interop/interop.cpp [hunk 1]
```diff
@@ -24,7 +24,7 @@ const QUIC_API_TABLE* MsQuic;
 HQUIC Registration;
 int EndpointIndex = -1;
 uint32_t TestCases = QuicTestFeatureAll;
-uint32_t WaitTimeoutMs = 5000;
+uint32_t WaitTimeoutMs = 10000;
```
Comment (nibanks) [inline src/tools/interop/interop.cpp:27]: This really necessary?

File: src/tools/interop/interop.cpp [hunk 2]
```diff
@@ -119,8 +119,8 @@ uint32_t CurrentThreadCount;
 
 uint16_t CustomPort = 0;
 
-const char* UrlPath = "/";
 bool CustomUrlPath = false;
+std::vector<const char*> Urls;
```
Comment (nibanks) [inline src/tools/interop/interop.cpp:123]: If you're using `vector`, maybe worth while to use `string` too?

File: src/tools/interop/interop.cpp [hunk 3]
```diff
@@ -1002,13 +1050,42 @@ main(
     TryGetValue(argc, argv, "timeout", &WaitTimeoutMs);
     TryGetValue(argc, argv, "version", &InitialVersion);
     TryGetValue(argc, argv, "port", &CustomPort);
-    if (TryGetValue(argc, argv, "urlpath", &UrlPath)) {
-        if (UrlPath[0] != '/' || strlen(UrlPath) == 1) {
-            printf("Invalid UrlPath! Must begin with '/'!\n");
-            Status = QUIC_STATUS_INVALID_PARAMETER;
-            goto Error;
+    //
+    // Get URLs from commandline.
```
Comment (nibanks) [inline src/tools/interop/interop.cpp:1054]: Maybe put the following code in a helper function?

**Review by nibanks — APPROVED** [review]

File: src/tools/interop/interop.cpp (hunk 1 above)
Comment (anrossi) [reply src/tools/interop/interop.cpp:27]: I was wondering if part of the reason the high RTT/loss tests fail is that the file doesn't finish transferring within 5 seconds. Some of those files are over 25 MB.

File: src/tools/interop/interop.cpp (hunk 2 above)
Comment (anrossi) [reply src/tools/interop/interop.cpp:123]: What benefit would there be, though?

Comment (nibanks) [reply src/tools/interop/interop.cpp:123]: you don't have to manage the memory?

File: src/tools/interop/interop.cpp (hunk 1 above)
Comment (nibanks) [reply src/tools/interop/interop.cpp:27]: Ok.

**Pull request merged**